File: server/src/main/java/com/example/server/dto/business/BusinessResponseDto.java
package com.example.server.dto.business;


import com.example.sep3.grpc.Location;
import com.example.server.dto.address.AddressDto;
import com.example.server.dto.offer.LocationDto;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class BusinessResponseDto {
    private String id;
    private String email;
    private String phoneNumber;
    private String hashedPassword;
    private AddressDto address;
    private String logo_path;
    private String cvr;
    private String businessName;
    private String role;
    private Double latitude;
    private Double longitude;
}

File: client/client/Dto/Business/BusinessResponseDto.cs
using Stripe;

namespace client.Dto.Business;

public class BusinessResponseDto
{
    public string Id { get; set; }
    public string Email { get; set; }
    public string PhoneNumber { get; set; }
    public AddressDto Address { get; set; }
    public string Logo_path { get; set; }
    public string Cvr { get; set; }
    public string BusinessName { get; set; }
    
    public string Role { get; set; }
    
    public Double Latitude { get; set; }
    public Double Longitude { get; set; }
}